ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -181,32 +183,71 @@ azure_add_reference(WT_STORAGE_SOURCE *storage_source)
     return 0;
 }
 
+// Flush a file locally to Azure.
```

Comment:
Hmm feels weird that we are flushing a file locally? I would say we "Flush given file to the Azure Blob sTorage.

Reply:
amended